interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,506 @@ components:
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - network
+              - broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['http://ussp.com/emergency']
+        endurance_minutes: 
+          type: number
+          description: A number to specify the endurance of the drone in minutes
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
+      - uas_serial_number
+      - operation_mode
+      - uas_class
+      - identification_technologies
+      - operator_id
+
+    OperationMode:
+      type: string
+      enum:
+        - vlos
+        - bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation
+
+    UASClass:
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
+      description: Specify the class of the UAS to be flown, the specifition matches EASA class identification label categories, for more information see EASA website
+
+    IdentificationTechnology:
+      type: string
+      enum:
+        - network
+        - broadcast
+      description: Specify an identification technology that will be used for this flight
+    
+    InjectFlightRequest:
+      type: object      
+      properties: 
+        injection_id: 
+          description: 'ID of this injection.  Remains the same regardless of the flight ID reported in the system.'
+          type: string
+          example: 'edb7695f-8737-4b9f-91f8-e2afbb333f41'
+
+        operational_intent:
+          description: A Operational Intent for a flight provided for the test
+          $ref: '#/components/schemas/OperationalIntentTestInjection'
+
+        flight_authorization:
```

Comment:
nit: Probably want consistent spelling between field and data type

Reply:
Thanks yes this is fixed now.